proto/v1/page.proto

```diff
@@ -32,3 +60,17 @@ message PlacePageRequest {
   // The category for which data needs to be fetched.
   string category = 4;
 }
+
+// Response to get place page info for a place.
+message PlacePageResponse {
```

Comment:
Hard to review the diffs... is the expectation that the proto fields exactly match v0 so that website code doesn't have to change? Or is there a corresponding website change?